devtools-ci-index.yaml

```diff
@@ -1667,6 +1667,24 @@
 
     <<: *eclipse-che-automation-template
 
+- job-template:
+    name: '{ci_project}-{git_repo}-che-integration-tests-rc-temporary'
```

Comment:
I think, that for this job we don't have to use "temporary". I think there's no harm in naming it with some final name, which will not change. For example:
`{ci_project}-{git_repo}-rc-integration-tests` -> which makes job name `devtools-che-rc-integration-tests`

Reply:
Done.